cmd/ro/gopass/gopass.go

```diff
@@ -91,11 +91,11 @@ func echoOff(fd []uintptr) (int, error) {
 
 // echoOn turns back on the terminal echo.
 func echoOn(fd []uintptr) {
-  // Turn on the terminal echo.
-  pid, e := syscall.ForkExec(sttyArg0, sttyArgvEOn, &syscall.ProcAttr{Dir: exec_cwdir, Files: fd})
-  if e == nil {
-    syscall.Wait4(pid, &ws, 0, nil)
-  }
+	// Turn on the terminal echo.
+	pid, e := syscall.ForkExec(sttyArg0, sttyArgvEOn, &syscall.ProcAttr{Dir: exec_cwdir, Files: fd})
```

Comment:
Ah; this file didn't have any functional changes; it just got caught up in my gofmt command. I'll remove it from the commit in the next push